common/djangoapps/student/models.py

```diff
@@ -505,6 +505,8 @@ class Meta(object):
     allow_certificate = models.BooleanField(default=1)
     bio = models.CharField(blank=True, null=True, max_length=3000, db_index=False)
     profile_image_uploaded_at = models.DateTimeField(null=True, blank=True)
+    phone_regex = RegexValidator(regex=r'^\+?1?\d*$', message="Phone number can only contain numbers.")
+    phone_number = models.CharField(validators=[phone_regex], blank=True, null=True, max_length=50)
```

Comment:
I remember when we were going over OEP-38 in one of the Data Guild meetings, it said models should be annotated for PII. It looks like this _does_ have the pii annotation, but do we have to add a new pii_type for "phone number"? OEP-38 points to OEP-30 here: https://github.com/edx/open-edx-proposals/blob/master/oeps/oep-0030-arch-pii-markup-and-auditing.rst.

Reply:
Added! Good catch.

Reply:
Added! Good catch.